simple_app/utils.py

```diff
@@ -474,6 +492,47 @@ def get_filters(db_file: str) -> pd.DataFrame:
     return phot_filters
 
 
+def checkdict(ref: str, refdict: dict, altref: str = 'altname', replace: bool = None) -> bool:
+    """
+    Checking dictionary, taken from splat
```

Comment:
Checking dictionary for what? This could benefit from a clearer docstring. I thought it was just checking if a key was in a dictionary regardless of case, but it looks a bit more complex than that.

Reply:
Agreed, this is a bit confusing. I've removed it (was just taken from splat originally)